src/firebaseConfig.js

```diff
@@ -0,0 +1,10 @@
+const firebaseConfig = {
+  apiKey: "AIzaSyCGI47hzOFkm2c5f2EzNM-7J1sx-8fqhs4",
+  authDomain: "vibeyard-1876b.firebaseapp.com",
+  projectId: "vibeyard-1876b",
+  storageBucket: "vibeyard-1876b.appspot.com",
+  messagingSenderId: "237733283385",
+  appId: "1:237733283385:web:697962f274ce55c8652fe8",
```

Comment:
Pass these parameters to your env file and call them here. Refrain from committing API Key details from next time.

Reply:
Thanks totally forgot about this